### ISSUE TITLE
Add to NPM repository

### ISSUE THREAD

**Issue by walterg2** [cloudspace/angular_devise#120]
Now that Bower has decided to just go into maintenance mode and are [recommending you move to using Yarn and Webpack](https://bower.io/blog/2017/how-to-migrate-away-from-bower/), would it be possible to push this out to NPM?

There is already a `package.json` file out there, so it would just be a matter of formally publishing the libarary.

**Comment by AleksandrLeontev** [comment]
yes, please. I would like to use yarn for this package, too

**Comment by mebibou** [comment]
yes it would be a lot better instead of doing `npm install git://github.com/cloudspace/angular_devise.git#v1.3.0`

**Comment by aboelnaga** [comment]
yes it would be very great
